### DISCUSSION TITLE
Selective Option Transfer and Save Presets

### DISCUSSION THREAD

**Discussion by leandrolima-nyc** [OrcaSlicer/OrcaSlicer#1224] [Ideas]
Currently, users face a challenge when working with comparing presets and saving profiles. Specifically, they lack the ability to transfer or save only selected options. This limitation forces users to either save or discard all options, which can be time-consuming and inefficient.

To address this issue, introducing the feature of selective transfer and save, similar to the functionality offered in Prusa Slicer, would significantly enhance the user experience. By allowing users to choose specific options for transfer or saving, they can streamline their workflow and tailor their configurations more precisely. This improvement not only saves time but also ensures that users have full control over their preferred settings, making the entire process more user-friendly and efficient.

![PrusaSlicer 2023-05-31 at 11 38 29PM@2x](https://github.com/SoftFever/OrcaSlicer/assets/70433062/0cd92bc6-b186-4fd8-a250-067fff7cad4f)

![PrusaSlicer 2023-05-31 at 10 53 04PM@2x](https://github.com/SoftFever/OrcaSlicer/assets/70433062/7ec0e030-e9df-41c9-80c6-6eecbb1aee6a)

![PrusaSlicer 2023-05-31 at 10 53 58PM@2x](https://github.com/SoftFever/OrcaSlicer/assets/70433062/96b88a0f-6ec9-4b0e-a883-486a45e0fc33)

**Comment by fr3ak2402** [comment]
@leandrolima-nyc 

Thank you for the feedback.

I have this function in development and am already testing this. The window will have a slightly different exterior and additional functionalities that will make the overview appear even simpler and more structured.

I will release the functionality in the next version of OrcaSlicer if everything goes well.

**Reply by leandrolima-nyc** [reply]
Wow, that's absolutely fantastic! I appreciate for your prompt response and it's truly admirable to see the level of effort you put into work. I'm eagerly anticipating the exciting enhancements that await us.

**Comment by julie777** [comment]
This dovetails into my discussion about a profile manager that would allow looking at a diff and moving some of the differences to the other side.  #1218

I think I missed mentioning that one of the problems with the existing compare, in addition to the inability to move settings to they other profile, is the inability to compare against the currently modified profile.
